daceml/onnx/op_implementations/pure_implementations.py

```diff
@@ -882,18 +738,9 @@ def Softplus(X, Y):
     Y[:] = np.log(1 + np.exp(X))
 
 
-@op_implementation(op="Sigmoid", name="pure")
-class PureSigmoid(ONNXForward):
-    @staticmethod
-    def forward(node: onnx_op.ONNXOp, state: SDFGState,
-                sdfg: SDFG) -> typing.Union[Node, SDFG]:
-        dtype = in_desc_with_name(node, state, sdfg, "X").dtype
-
-        def prog(X, Y):
-            Y[:] = dace.elementwise(lambda x: dtype(1) / (dtype(1) + exp(-x)),
-                                    X)
-
-        return program_for_node(prog, sdfg, state, node)
+@python_pure_op_implementation(dtype=lambda X: X.dtype)
+def Sigmoid(X, Y):
+    Y[:] = dace.elementwise(lambda x: dtype(1) / (dtype(1) + exp(-x)), X)
```

Comment:
for the other ones you used a string. Why did you not use one here?

Reply:
I don't remember, there must have been some bug back when I wrote it